src/payment/create-payment-strategy-registry.ts

```diff
@@ -345,14 +346,19 @@ export default function createPaymentStrategyRegistry(
         )
     );
 
+    const LOADING_INDICATOR_STYLES = {
```

Comment:
You can just pass the properties directly instead of using a temporary variable e.g.:
```ts
new LoadingIndicator({ styles: { backgroundColor: 'black' } })
```